doc/src/sgml/config.sgml

```diff
@@ -2145,6 +2145,34 @@ include_dir 'conf.d'
       </listitem>
      </varlistentry>
 
+     <varlistentry id="guc-max-total-memory" xreflabel="max_total_memory">
+      <term><varname>max_total_memory</varname> (<type>integer</type>)
+      <indexterm>
+       <primary><varname>max_total_memory</varname> configuration parameter</primary>
+      </indexterm>
+      </term>
+      <listitem>
+       <para>
+        Specifies a limit to the total amount of memory (MB) a database server
+        can allocate. (i.e. this is not a per user or per backend limit).
+        If unset or set to 0, it is disabled. max_total_memory must be larger than
+        shared_memory_size and should include an additional 2MB per connection.
```

Comment:
I don't like how this is saying "should include" ... maybe "at least include" or similar, or we could just drop that part of the sentence entirely.